doc/docs.md

```diff
@@ -190,7 +190,7 @@ by using any of the following commands in a terminal:
    		* [Trace](#trace)
     * [Memory-unsafe code](#memory-unsafe-code)
     * [Structs with reference fields](#structs-with-reference-fields)
-    * [sizeof and __offsetof](#sizeof-and-__offsetof)
+    * [sizeof, alignof and __offsetof](#sizeof-alignof-and-__offsetof)
```

Comment:
Why `alignof` instead of `__alignof` ?

@medvednikov what should be the name of this?

Reply:
I can not decide, but I do think that we should prefer one of them, while it is not used yet a lot.